lib/twitter-ads/resources/analytics.rb

```diff
@@ -209,18 +225,46 @@ def fetch_async_job_data(data_url)
         end
       end
 
+      # Retrieve details about which entities' analytics metrics
+      # have changed in a given time period.
+      #
+      # @example
+      #   time = Time.now
+      #   utc_offset = '+09:00'
+      #   start_time = time - (60 * 60 * 24) # -1 day
+      #   end_time = time
+      #   active_entities = TwitterAds::LineItem.active_entities(
+      #     account,
+      #     line_item_ids: %w(exrfs),
+      #     start_time: start_time,
+      #     end_time:   end_time,
+      #     utc_offset: utc_offset,
+      #     granularity: :day)
+      #
+      # @param account [Account] The Account object instance.
+      # @param entity [String] The entity type to retrieve data for.
```

Comment:
It sounds like with this implementation the `entity` is inferred by the calling class, correct? If so, should we update these docs to reflect the actual params available?

Reply:
@tushdante Thanks for asking that, yes it was but I fixed it in this commit https://github.com/twitterdev/twitter-ruby-ads-sdk/pull/210/commits/6704ed5cf2ad05ae841d90e3ca7fdc157f97e327 so now it can be called directly with specifying `entity` parameter:

```ruby
TwitterAds::Analytics.active_entities( [..], entity: 'LINE_ITEM' )
```

Reply:
Oh gotcha, sounds good!